e2echart/e2e-chart-template.html

```diff
@@ -464,6 +494,12 @@ <h5 class="modal-title">Resource</h5>
         timelineGroups.push({group: "endpoint-availability", data: []})
         createTimelineData(disruptionValue, timelineGroups[timelineGroups.length - 1].data, eventIntervals, isEndpointConnectivity, regex)
 
+        timelineGroups.push({group: "apiserver-disruption", data: []})
+        createTimelineData(apiserverDisruptionValue, timelineGroups[timelineGroups.length - 1].data, eventIntervals, isNewDisruptionActivity, regex)
+
+        timelineGroups.push({group: "apiserver-shutdown", data: []})
```

Comment:
We are in-between moving this template to a better location. David has another implementation in the same directory as this one with some improvements, but it's long term goal is now sippy which will allow us to properly link other engineers directly to what we're looking at (filters and all), as well as roll out improvements which can take effect for past jobs. (as we're not using a static artifact) You can see the new link to Sippy intervals in prow "Debug Tools" spyglass panel, and the Intevals button.

The code is here: https://github.com/openshift/sippy/blob/master/sippy-ng/src/prow_job_runs/ProwJobRun.js

We can port it though if you'd rather just stay focused in origin.